de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5521740_sys_gh5208_Change_Age_Attribute_Type_To_List_And_Add_Values.sql

```diff
@@ -0,0 +1,70 @@
+INSERT INTO M_AttributeValue (AD_Client_ID,AD_Org_ID,Created,CreatedBy,IsActive,IsNullFieldValue,M_Attribute_ID,M_AttributeValue_ID,Name,Updated,UpdatedBy,Value) VALUES (0,0,TO_TIMESTAMP('2019-05-16 14:27:29','YYYY-MM-DD HH24:MI:SS'),100,'Y','N',540042,540026,'6',TO_TIMESTAMP('2019-05-16 14:27:29','YYYY-MM-DD HH24:MI:SS'),100,'6')
+;
+
+-- 2019-05-16T14:27:33.336
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO M_AttributeValue (AD_Client_ID,AD_Org_ID,Created,CreatedBy,IsActive,IsNullFieldValue,M_Attribute_ID,M_AttributeValue_ID,Name,Updated,UpdatedBy,Value) VALUES (0,0,TO_TIMESTAMP('2019-05-16 14:27:33','YYYY-MM-DD HH24:MI:SS'),100,'Y','N',540042,540027,'7',TO_TIMESTAMP('2019-05-16 14:27:33','YYYY-MM-DD HH24:MI:SS'),100,'7')
+;
+
+-- 2019-05-16T14:27:35.697
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO M_AttributeValue (AD_Client_ID,AD_Org_ID,Created,CreatedBy,IsActive,IsNullFieldValue,M_Attribute_ID,M_AttributeValue_ID,Name,Updated,UpdatedBy,Value) VALUES (0,0,TO_TIMESTAMP('2019-05-16 14:27:35','YYYY-MM-DD HH24:MI:SS'),100,'Y','N',540042,540028,'8',TO_TIMESTAMP('2019-05-16 14:27:35','YYYY-MM-DD HH24:MI:SS'),100,'8')
+;
+
+-- 2019-05-16T14:27:39.103
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO M_AttributeValue (AD_Client_ID,AD_Org_ID,Created,CreatedBy,IsActive,IsNullFieldValue,M_Attribute_ID,M_AttributeValue_ID,Name,Updated,UpdatedBy,Value) VALUES (0,0,TO_TIMESTAMP('2019-05-16 14:27:38','YYYY-MM-DD HH24:MI:SS'),100,'Y','N',540042,540029,'9',TO_TIMESTAMP('2019-05-16 14:27:38','YYYY-MM-DD HH24:MI:SS'),100,'9')
+;
+
+-- 2019-05-16T14:27:43.630
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO M_AttributeValue (AD_Client_ID,AD_Org_ID,Created,CreatedBy,IsActive,IsNullFieldValue,M_Attribute_ID,M_AttributeValue_ID,Name,Updated,UpdatedBy,Value) VALUES (0,0,TO_TIMESTAMP('2019-05-16 14:27:43','YYYY-MM-DD HH24:MI:SS'),100,'Y','N',540042,540030,'10',TO_TIMESTAMP('2019-05-16 14:27:43','YYYY-MM-DD HH24:MI:SS'),100,'10')
+;
+
+-- 2019-05-16T14:27:46.175
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO M_AttributeValue (AD_Client_ID,AD_Org_ID,Created,CreatedBy,IsActive,IsNullFieldValue,M_Attribute_ID,M_AttributeValue_ID,Name,Updated,UpdatedBy,Value) VALUES (0,0,TO_TIMESTAMP('2019-05-16 14:27:46','YYYY-MM-DD HH24:MI:SS'),100,'Y','N',540042,540031,'11',TO_TIMESTAMP('2019-05-16 14:27:46','YYYY-MM-DD HH24:MI:SS'),100,'11')
+;
+
+-- 2019-05-16T14:27:50.593
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO M_AttributeValue (AD_Client_ID,AD_Org_ID,Created,CreatedBy,IsActive,IsNullFieldValue,M_Attribute_ID,M_AttributeValue_ID,Name,Updated,UpdatedBy,Value) VALUES (0,0,TO_TIMESTAMP('2019-05-16 14:27:50','YYYY-MM-DD HH24:MI:SS'),100,'Y','N',540042,540032,'12',TO_TIMESTAMP('2019-05-16 14:27:50','YYYY-MM-DD HH24:MI:SS'),100,'12')
```

Comment:
What happens if we run out of "age" values, i.e. if the first piece of cheese turns 13 months old?

Reply:
Working on it now, but in the end there will be a hard cap at 24 months. Everything that is over 24 months will simply have 24 displayed.